src/chi/components/card/card.scss

```diff
@@ -124,7 +124,44 @@
       &.-noBorder {
         box-shadow: none;
       }
+    }
+
+    &.-empty {
+      background-color: transparent;
+      border: dashed rgba(set-color(black), 0.12);
+      transition: all 0.2s;
+
+      & .a-card__content {
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        margin: 1rem;
+
+        & .a-icon {
+          color: set-color(grey, 60);
```

Comment:
@davidojedacl @jllr We should not be defining colors here as developers will be unable to override. Please use color utilities: `-text--muted`

Reply:
ok perfect